solver/src/orderbook.rs

```diff
@@ -16,7 +16,7 @@ impl OrderBookApi {
     }
 
     pub async fn get_orders(&self) -> reqwest::Result<Vec<Order>> {
-        const PATH: &str = "/api/v1/orders";
+        const PATH: &str = "/api/v1/solvable_orders";
```

Comment:
Doesn't this change the request path for orders too? Shouldn't there just be a new method introduced here `get_solvable_orders` that introduces this new path and not overwrite the existing one?

Reply:
This is the path we use to query orders in the solver so we ony need the new path.